libs/shared/src/types/clerk/clerk.user.types.ts

```diff
@@ -0,0 +1,93 @@
+import { User as ClerkUser } from '@clerk/clerk-sdk-node';
+import { ClerkPaginationRequest } from '@clerk/types';
+import { IServicesHashes } from '../../entities/user';
+import { JobTitleEnum } from '../organization';
+
+export type User = ClerkUser & UserMetadataParams;
+
+export type UpdateUserParams = {
```

Comment:
Can we get these types from Clerk?

Reply:
We can report that, but try reusing types as much as possible. Note that, this is not a blocker for merging.